test/csv/parse/test_strip.rb

```diff
@@ -80,4 +80,33 @@ def test_do_not_strip_crlf
                            %Q{"a" ,"b " \r\n},
                            strip: true))
   end
+
+  def test_raise_error_if_col_sep_is_tab_and_strip_is_true
```

Comment:
Can we simplify this like the following?

```suggestion
  def test_col_sep_incompatible_true
```

This file is for `:strip`. So we don't need to mention `strip` in this file.